### DISCUSSION TITLE
[공지사항] 2021-01-23 ~ 2021-01-29

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#164] [00_공지사항]
여러분 안녕하세요!

멘토 유혁준, 강형구입니다.

## 과제

1. 쥬피터파일말고 마크다운 파일 생성후, Issue 정리하고 push
2. 멘티들 calculator 리뷰하고 마크다운 파일에 같이 정리해서 push하기
3. 형구형 calculator 분석하고 코드에 주석달아서 자세히 분석하기
4. (선택) 프로젝트하면서 공부한 것들 Notion에 정리하기

이번에 push할 파일은 issue정리 및 calculator 리뷰정리한 마크다운, 형구형 calculator 파일 이렇게 2개 업로드하시면 됩니다.

이번 과제는 경량화하였고, 프로젝트에 집중하라고 과제를 소규모로 하였습니다.

**Comment by lsj0822** [comment]
확인했습니다

**Comment by dlrudgnsdlek** [comment]
확인했습니다

**Comment by jiwoo4424** [comment]
확인했습니다

**Comment by NA-DK** [comment]
확인했습니다!
